### DISCUSSION TITLE
Is there privacy mode = black screen support?

### DISCUSSION THREAD

**Discussion by naddel91** [rustdesk/rustdesk#1462] [General]
Hi,

I am currently looking for an alternative to TeamViewer (expensive and almost no support) or AnyViewer (offline all the time), but I need Black Screen support on client systems. Does RustDesk support that?

**Comment by naddel91** [comment]
I guess not :(

**Reply by Loremipsumdolorsitametconsetetur** [reply]
Check out privacy mode 2, also explained here https://www.reddit.com/r/rustdesk/comments/1gf5goq/privacy_mode_v2/?show=original
